cmd/mrconvert.cpp

```diff
@@ -259,9 +260,11 @@ inline vector<int> set_header (Header& header, const ImageType& input)
 
   opt = get_options ("vox");
   if (opt.size()) {
-    vector<default_type> vox = opt[0][0];
+    vector<default_type> vox = parse_floats (opt[0][0]);
```

Comment:
Not a fan of implicit type-casting...? 😉

Reply:
Generally yes; but when it's possible to provide either a single value *or* a comma-separated list to the same option, I'd rather be more explicit than less.

Plus it was like that in the `mrresize` code :man_shrugging:

Reply:
Oh well... It is clearer that way 👍